src/docs/product/performance/distributed-tracing.mdx

```diff
@@ -4,6 +4,7 @@ sidebar_order: 3
 redirect_from:
   - /performance/distributed-tracing/
   - /performance-monitoring/distributed-tracing/
+description: "Learn more about distributed tracing and how Sentry tracks your software performance, measuring things like throughput and latency, and can also display the impact of errors across multiple systems."
```

Comment:
seems weird

Reply:
Which part? 😄 I agree it needs some smoothing out.

Reply:
```suggestion
description: "Learn more about distributed tracing and how Sentry tracks your software performance, measure things like throughput and latency, and can also display the impact of errors across multiple systems."
```

I think this is the correction it needs.